content/news/2024-02-03-bevy-0.13/index.md

```diff
@@ -0,0 +1,1676 @@
+
+The new asset processing pipeline, using the `LoadTransformAndSave` `Process` implementation, has five stages:
+
+1. An `AssetReader` read some asset source (filesystem, http, etc) and gets the byte data of an asset.
+2. An `AssetLoader` reads the byte data and converts it to a bevy `Asset`.
+3. An `AssetTransformer` takes an asset and transforms it in some way.
+4. An `AssetSaver` takes a bevy `Asset` and converts it back into byte data.
+5. An `AssetWriter` then writes the asset byte data back to the asset source.
+
+In addition to having better code reusability, this change encorages writing `AssetSaver`s for various common asset types, which could be used to add runtime asset saving functionality to the `AssetServer`.
+
+The previous `LoadAndSave` `Process` implementation still exists, as there are some cases where an asset transformation step is unnecessary, such as when saving assets into a compressed format.
+
+See the [Asset Processing Example](<https://github.com/bevyengine/bevy/blob/main/examples/asset/processing/asset_processing.rs>) for a more detailed look into how to use `LoadTransformAndSave` to process a custom asset.
+
+## Entity Optimizations
+
+<div class="release-feature-authors">authors: @Bluefinger, @notverymoe, @scottmcm, @bushrat011899, @james7132</div>
+
+`Entity` (Bevy's 64-bit unique identifier for enitities) received a number of changes this cycle, combining laying some more groundwork for relations alongside _related_, and nice to have, performance optimizations. The work here involved a lot of deep-diving into compiler codegen/assembly output, with running lots of benchmarks and testing in order to ensure all changes didn't cause breakages or major problems. Although the work here was dealing with mostly _safe_ code, there were lots of underlying assumptions being changed that could have impacted code elsewhere. This was the most "micro-optimization" oriented set of changes in Bevy 0.13.
+
+* [#9797]: created a unified identifier type, paving the path for us to use the same fast, complex code in both our `Entity` type and the much-awaited relations
+* [#9907]: allowed us to store `Option<Entity>` in the same number of bits as `Entity`, by changing the layout of our Entity type to reserve exactly one `u64` value for the `None` variant
+* [#10519]: swapped us to a manually crafted `PartialEq` and `Hash` implementation for `Entity` to improve speed and save instructions in our hot loops
+* [#10558]: took the same approach, and further optimized our `PartialOrd` and `Ord` implementations!
```

Comment:
```suggestion
* [#10558]: combined the approach of [#9907] and [#10519] to optimize `Entity`'s layout further , and optimized our `PartialOrd` and `Ord` implementations!
```
This is a nit, but I think it better describes/summarises the follow-up and what the work was built upon.